tic_tac_toe/game.py

```diff
@@ -87,18 +128,54 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
-
-
+    if not _position_is_valid(position):
+        raise InvalidMovement("Position out of range.")
+
+    if _board_is_full(game['board']) or game['winner'] is not None:
```

Comment:
I'm rethinking this because of the note on the second point [here](https://www.python.org/dev/peps/pep-0008/#programming-recommendations).  I guess the user could technically enter a player in game setup that would evaluate to `False` in a boolean context, though that would probably make other parts of the game ugly.